Sources/GIF/code/GIFEncoder.swift

```diff
@@ -30,10 +32,54 @@ struct GIFEncoder {
         }
 
         // TODO: Encode comment extensions
-
+      
+      
+//      let start = CACurrentMediaTime()
+      #if true
```

Comment:
We should remove the redundant branches here, along with the commented-out benchmarking checkpoints.